regicide_game/player.py

```diff
@@ -0,0 +1,65 @@
+import random
+from typing import List
+
+from regicide_game.card import Card
+
+
+class Player:
+    hand: List(Card)
```

Comment:
I actually do not know what this notation does, but I thought it needs to be written as `List[Card]`
```suggestion
    hand: List[Card]
```